ops/dev/restore-pacs-from-archives.ps1

```
@@ -0,0 +1,228 @@
+  SAFE: Does not touch existing containers unless -RemoveExistingContainer is set.
+  SECURE: SA password must be provided via parameter. Never generated or echoed.
+
+  Usage:
+    pwsh -NoProfile -File ops/dev/restore-pacs-from-archives.ps1 `
+      -SaPassword "YourStr0ng!Pass" `
+      -RemoveExistingContainer -RemoveExistingVolumes
+#>
+
+param(
+  [Parameter(Mandatory=$true)]
+  [string]$SaPassword,
```

PSScriptAnalyzer (1.24.0)

[warning] Parameter '$SaPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $SaPassword type to SecureString; Set $SaPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)